### DISCUSSION TITLE
Updating Error ModuleNotFound: 'ezdxf'

### DISCUSSION THREAD

**Discussion by OwieSetya** [cnr-isti-vclab/TagLab#170] [Q&A] [answered]
Hello, I'm trying to update TagLab and i was download the latest version file. After the update was done, I tried running the program, but there was an error saying ModuleNotFoundError: No module name 'ezdxf', and what should i do to fix this?. Here the screenshot about the trouble.

![Screenshot 2024-11-29 222458](https://github.com/user-attachments/assets/94032a36-d6b6-46fd-987b-32b451ca70ff)

I'm very glad for your help. Thanks.

**Comment by maxcorsini** [comment, marked as answer]
Run install.py and retry

Il Ven 29 Nov 2024, 16:47 OwieSetya ***@***.***> ha scritto:

**Reply by OwieSetya** [reply]
it works, thank you very much!

**Reply by Chris-Cooney** [reply]
I am having the same issue and reinstalling is not solving it

**Reply by maxcorsini** [reply]
Dear Chris,

re-run install.py after the update should work, but if it is not your case let's try to install manually ezdxf by writing the command

pip install ezdxf 

in your shell.
